### PR TITLE
Theme docs/blogs with Tailwind

### PR DESCRIPTION
**Pull request by og118**
- Setup Tailwind css for blogs and docs page
- Changed the theme for the docs/blogs page and tried to match it with the rest of the app.

Screenshots of the new docs/blog page:
Light Mode:
image
Dark Mode:
image

The colors do not match perfectly, and I needed help deciding the color palette for tailwind and starlight (added a comment for the same) @vincanger 

Addresses #7

### CODE REVIEW

File: blog/tailwind.config.mjs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const yellow = colors.yellow
+const gray = colors.gray
+
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ["./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}"],
+  theme: {
+    extend: {
+      colors: {
+        accent: yellow, gray
```
Comment (og118) [inline blog/tailwind.config.mjs:13]: Here, I needed input on what color palette should be used.

As per starlight [docs](https://starlight.astro.build/guides/css-and-tailwind/#color-theme-editor),
`accent` determines the colors of links, active nav items, etc.
`gray` determines the color of background in both dark and light modes

Currently, I assigned them to pure `yellow` and `grey` palletes that come along with tailwind.
```
const yellow: {
    '50': "#fefce8";
    '100': "#fef9c3";
    '200': "#fef08a";
    '300': "#fde047";
    '400': "#facc15";
    '500': "#eab308";
    '600': "#ca8a04";
    '700': "#a16207";
    '800': "#854d0e";
    '900': "#713f12";
    '950': "#422006";
}
```
and, 
```
gray: {
    '50': "#f9fafb";
    '100': "#f3f4f6";
    '200': "#e5e7eb";
    '300': "#d1d5db";
    '400': "#9ca3af";
    '500': "#6b7280";
    '600': "#4b5563";
    '700': "#374151";
    '800': "#1f2937";
    '900': "#111827";
    '950': "#030712";
}
```

**Comment by Martinsos** [comment]
Wow nice @og118 !

@vincanger is the best person to take a look -> it might take him a couple of days to take a look because he is on a trip though!

File: blog/tailwind.config.mjs (hunk 1 above)
Comment (vincanger) [reply blog/tailwind.config.mjs:13]: checked it out @og118 and it looks great! let's go with the defaults 🤝

**Comment by og118** [comment]
Hi @vincanger, the PR shows a whopping 1240 lines of added code because of the autogenerated `package.lock`, should I remove it from the changes?

**Comment by vincanger** [comment]
That’s fine @og118 ! 

I’ll check this PR out tomorrow when I’m back at my laptop but from what I can see now it looks ready to merge. Great work and thanks so much for your contribution 🙏

**Comment by vincanger** [comment]
Oh yes, looking sweet. Thanks so much for your contribution here @og118! Much appreciated

**Review by vincanger — APPROVED** [review]

**Pull request merged**